app/javascript/articles/LoadingArticle.jsx

```diff
@@ -0,0 +1,16 @@
+import { h } from 'preact';
+
+export const LoadingArticle = () => (
```

Comment:
This was pulled from https://github.com/thepracticaldev/dev.to/blob/master/app/views/stories/_main_stories_feed.html.erb#L15 to create the Preact component.